src/components/CTASecondary.js

```diff
@@ -29,35 +28,34 @@ const CTASecondary = ({ classes, className }) => (
 export default withStyles((theme) => ({
   root: {
     display: "flex",
-    justifyContent: "center",
     backgroundColor: "#55C3BA",
     padding: "40px 12px",
-
-    [theme.breakpoints[900]]: {
-      padding: "60px 12px",
-    },
+    justifyContent: "center",
+    color: "#FFF",
```

Comment:
I see `#FFF` and `#FFF !important` on some of the selectors below– seems redundant.

I normally expect to see `color` applied directly on the element that contains the text. If we're going to be overriding *all* text in the container, it's fine to have it in root, but then I wouldn't expect it to be repeated on `heading`, `ctaRight`, etc.